daprdocs/content/en/reference/errors/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+type: docs
+title: Dapr errors
+linkTitle: "Dapr errors"
+weight: 700
+description: "Information on Dapr errors and how to handle them"
+---
+
+## Dapr Error Handling: Understanding the Error Models
+
+Initially, Dapr followed the standard gRPC error model. However, to provide more detailed and informative error messages, Dapr is gradually transitioning to a richer error model as defined by gRPC. 
+
+{{% alert title="Note" color="primary" %}}
+Not all Dapr errors have been converted to the richer gRPC error model.
+{{% /alert %}}
+
+### Standard gRPC Error Model
+
+The [standard gRPC error model](https://grpc.io/docs/guides/error/#standard-error-model) is a straightforward approach to error reporting gRPC. Each error response includes an error code and an error message. The error codes are standardized and reflect common error conditions. 
+
+**Example of a Standard gRPC Error Response:**
```

Comment:
```suggestion
**Example of a standard gRPC Error response:**
```

Reply:
Since this is a named standard, does it make sense to keep it capitalised?